src/main/python/NOTICE

```diff
@@ -0,0 +1,8 @@
+SystemDS
+Copyright 2018 Graz University of Technology
```

Comment:
update

Reply:
should I replace with `[2018-2020]` or `2020`. The `NOTICE` file in the root dir should then also be updated I guess?

Reply:
I think that we can update the notice to say `[2018-2020]` and then write 2020 in these files.